config/lib/axios.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+// create an axios instance with defaults
+export default axios.create({
+  transformResponse: [({ data }) => data] // return only request body
+});
```

Comment:
Untested but we might need something like this:

```js
// Add a response interceptor
const axiosInstance = axios.create();

axiosInstance.interceptors.response.use((response) => {
    return Promise.resolve(response.data); // or just `return response.data;`?
  }, (error) => {
    // In Angular, we would emit a global event at this point and the application knows what to do with it (e.g. it would show "app is offline" or throw you to the login page.
    return Promise.reject(error);
});

export default axiosInstance;
```